src/utils/track.js

```diff
@@ -39,6 +39,15 @@ export default class Tracker {
     }
   }
 
+  trackComponent(type, properties) {
+    switch(type) {
+      case 'product':
+        return this.track('Viewed Product', properties);
+      case 'collection':
+        return this.track('Viewed Collection', propreties);
```

Comment:
It's `Viewed Collections` here
https://github.com/Shopify/widget-framework/blob/master/app/assets/javascripts/framework/lib/tracking.js.coffee#L118